packages/atproto_codegen/models/generator.py

```diff
@@ -139,7 +140,7 @@ def _get_optional_typehint(type_hint: str, *, optional: bool) -> str:
 
 
 def _get_str_enum_typehint(nsid: NSID, field_type_def: models.LexString, *, optional: bool) -> str:
-    values = field_type_def.known_values or field_type_def.enum
+    values = field_type_def.known_values or field_type_def.enum or []
```

Comment:
i can revert it buts its because both variables in this expression are optional, which appears to come right from `LexString` field definitions
```python
field_type_def.known_values or field_type_def.enum
```